mathesar/serializers.py

```diff
@@ -67,6 +70,11 @@ def get_records(self, obj):
         else:
             return None
 
+    def validate(self, data):
```

Comment:
Since this only touches a single field, `data_files`, I think it should be in a `validate_data_files` method.